src/mesh/mesh_adaptation.cpp

```diff
@@ -39,17 +42,27 @@ void MeshAdaptation<dim,real,MeshType>::fixed_fraction_isotropic_refinement_and_
     {
         dealii::GridRefinement::refine_and_coarsen_fixed_number(*(dg->high_order_grid->triangulation),
                                                                 cellwise_errors,
-                                                                refinement_fraction,
-                                                                coarsening_fraction);
+                                                                h_refine_fraction,
+                                                                h_coarsen_fraction);
     } 
     else 
     {
         dealii::parallel::distributed::GridRefinement::refine_and_coarsen_fixed_number(*(dg->high_order_grid->triangulation),
                                                                                         cellwise_errors,
-                                                                                        refinement_fraction,
-                                                                                        coarsening_fraction);
+                                                                                        h_refine_fraction,
+                                                                                        h_coarsen_fraction);
     }
 
+    // Currently, the error indicator to flag p_adaptation is the same as h_adaptation, but it will likely change in future.
```

Comment:
should we open an issue for this?

Reply:
Hmm, I am not sure if it's an issue. There are several ways to decide whether to perform h or p adaptation and it will likely depend on my research later. Currently, it's just a framework put in place to ensure p-adaptation works.

Reply:
Sounds good